bluez-generated/README.md

```diff
@@ -1,5 +1,8 @@
 # Generated bindings for BlueZ
 
+[![crates.io page](https://img.shields.io/crates/v/bluez-generated.svg)](https://crates.io/crates/bluez-generated)
+[![docs.rs page](https://docs.rs/bluez-generated/badge.svg)](https://docs.rs/bluez-generated)
```

Comment:
Hrm. I guess I should do something to say least make the readme show up in docs. rs.

In theory, I could generate code from the bluez docs, any have doc strings, but that sounds like a lot of work.